actions/transfer-case.vbs

```diff
@@ -138,466 +178,296 @@ EMReadScreen mail_addr_line_four, 43, 12, 27
-		IF forms_to_swkr = "Y" THEN EMWriteScreen "x", swkr_row, 10     'If forms_to_arep was "Y" (see above) it puts an X on the row ALTREP was found.
-		TRANSMIT                                       'TRANSMITs to start the memo writing process
-		PF4'save and exit
-	End if
-
-    'MNPrairie Bank Support - MNPrairie Bank cases all go to Steele (county code 74)'s ICT transfer.
-    'Agencies in the MNPrairie Bank are Dodge (county code 20), Steele (county code 74), and Waseca (county code 81)
-	IF servicing_worker = "X120ICT" OR servicing_worker = "X181ICT" THEN servicing_worker = "X174ICT"
-
-    'Using move date to determine CRF change date.
-	cfr_date = dateadd("M", 1, move_date)
-	cfr_date = datepart("M", cfr_date) & "/01/" & datepart("YYYY", cfr_date)
-	cfr_date = dateadd("M", 2, cfr_date)
-	cfr_month = datepart("M", cfr_date)
-	IF len(cfr_month) <> 2 THEN cfr_month = "0" & cfr_month
-	cfr_year = datepart("YYYY", cfr_date)
-	cfr_year = right(cfr_year, 2)
-
-	'----------The case note of the reason for the XFER----------
+	LOOP UNTIL are_we_passworded_out = False					'loops until user passwords back in
+
+    'SENDING a SPEC/MEMO - this happens before the case note, and transfer -  we overwrite the information
+    '----------Sending the Client a SPEC/MEMO notifying them of the details of the transfer----------
+
+    Call start_a_new_spec_memo(memo_opened, True, forms_to_arep, forms_to_swkr, send_to_other, other_name, other_street, other_city, other_state, other_zip, True)    		'Writes the appt letter into the MEMO.
+    Call write_variable_in_SPEC_MEMO("Your case has been transferred. Your new agency/worker is: " & worker_agency_name & "")
+    Call write_variable_in_SPEC_MEMO("If you have any questions, or to send in requested proofs,")
+    Call write_variable_in_SPEC_MEMO("please direct all communications to the agency listed.")
+    Call write_variable_in_SPEC_MEMO(worker_agency_name)
+    Call write_variable_in_SPEC_MEMO(mail_addr_line_one)
+    Call write_variable_in_SPEC_MEMO(mail_addr_line_two)
+    Call write_variable_in_SPEC_MEMO(mail_addr_line_three)
+    Call write_variable_in_SPEC_MEMO(mail_addr_line_four)
+    Call write_variable_in_SPEC_MEMO(worker_agency_phone)
+    Call write_variable_in_SPEC_MEMO("Domestic violence brochures are available at https://edocs.dhs.state.mn.us/lfserver/Public/DHS-3477-ENG.")
+    Call write_variable_in_SPEC_MEMO("You can also request a paper copy.  Auth: 7CFR 273.2(e)(3).")
+
+    IF forms_to_arep = "Y" THEN EMWriteScreen "x", arep_row, 10     'If forms_to_arep was "Y" (see above) it puts an X on the row ALTREP was found.
+    IF forms_to_swkr = "Y" THEN EMWriteScreen "x", swkr_row, 10     'If forms_to_arep was "Y" (see above) it puts an X on the row ALTREP was found.
+    TRANSMIT                                       'TRANSMITs to start the memo writing process
```

Comment:
This transmit shouldn't be needed here. I'm not sure what this would do.

Reply:
Removed unneeded lines of code